packages/sources/blockchain.com/src/index.ts

```diff
@@ -1,10 +1,13 @@
 import { expose } from '@chainlink/ea-bootstrap'
 import { endpointSelector, makeExecute } from './adapter'
 import { makeConfig, NAME } from './config'
+import rateLimit from './config/limits.json'
+
+const adapterContext = { name: NAME, rateLimit }
 
```

Comment:
Since we are changing this line, could we modify this to be standard with a standard export style?
```suggestion
const { server } = expose(adapterContext, makeExecute(), undefined, endpointSelector)
```